src/main/flight/feedforward.c

```diff
@@ -93,22 +93,25 @@ FAST_CODE_NOINLINE float feedforwardApply(int axis, bool newRcFrame, feedforward
 
         // interpolate setpoint if necessary
         if (rcCommandDelta == 0.0f) {
-            if (prevDuplicatePacket[axis] == false) {
-                // first duplicate after movement, interpolate setpoint and use previous acceleration
-                // don't interpolate if sticks close to centre or max, interpolate jitter signals less than larger ones
-                if (setpointPercent > 0.02f && setpointPercent < 0.95f) {
-                    // setpoint interpolation includes previous acceleration and attenuation
-                    setpoint = prevSetpoint[axis] + (prevSetpointSpeed[axis] + prevAcceleration[axis] * jitterAttenuator ) * rxInterval * jitterAttenuator;
-                    // recalculate speed and acceleration
-                    setpointSpeed = (setpoint - prevSetpoint[axis]) * rxRate;
-                }
+            // duplicate packet data on this axis
+            if (goodPacketCount[axis] == 2 && setpointPercent < 0.95f) {
```

Comment:
No need for `setpointPercenct > 0.02f` anymore?
As I understand correctly you now only permit interpolation when at least 2 valid packets have arrived instead of acting on the first duplicate.
The comment is stating two or more preceding valid packets but the counter never goes beyond 2.

Reply:
Yes, the original non-interpolation close to centre sticks was intended to counter this problem when it occurred from jitter around centre stick.  The new solution is much better because it applies regardless of stick position, and achieves the same thing.

As to the counter, I stop it at 2.  Hence the count loop will hit 2 after two (or more) valid packets, and stay there, not being incremented any more, returning '2' until it is reset to zero by a duplicate. The counter shouldn't ever be able to get >2.